src/components/DocumentLayouts.js

```diff
@@ -12,20 +12,36 @@ import {
 import MainLayout from './LayoutTypes/MainLayout'
 
 // import styles from './styles'
-import { BulletList as BulletListStyles, ColumnView as ColumnViewStyles, RowView as RowViewStyles } from './LayoutTypes/LayoutStyles'
-//import BulletListStyles from './LayoutStyles/BulletList'
-//import RowViewStyles from './LayoutStyles/RowViewStyles'
-//import ColumnViewStyles from './LayoutStyles/ColumnView'
+
+// import BulletListStyles from './LayoutStyles/BulletList'
+// import RowViewStyles from './LayoutStyles/RowViewStyles'
+// import ColumnViewStyles from './LayoutStyles/ColumnView'
+
+import { BulletList, ColumnView, RowView } from './LayoutTypes/LayoutStyles'
```

Comment:
Ok,  it looks similar but at '/LayoutStyles/' we have 3 files. You cannot import 3 different components from 3 different files without "....". Find it out